core/templates/dev/head/domain/exploration/FallbackObjectFactory.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating new frontend instances of Fallback
+ * domain objects.
+ */
+
+oppia.factory('FallbackObjectFactory', [function() {
+  var Fallback = function(triggerBackendDict, outcomeBackendDict) {
+    this.trigger = triggerBackendDict;
+    this.outcome = outcomeBackendDict;
+  };
+
+  Fallback.prototype.toBackendDict = function() {
+    return {
+      trigger: this.trigger,
+      outcome: this.outcome
+    };
+  };
+
+  Fallback.create = function(fallbackDict) {
```

Comment:
Yeah, that's what I was thinking. Sounds like we agree!

'correct' is a new property that was introduced mainly for the simple editor and for the repurposing of the exploration structure to be compatible with the new "questions" framework, so it is indeed always false for now.